client/src/components/Firebase/firebase.js

```diff
@@ -69,11 +69,24 @@ class Firebase {
       const orgDocument = await this.firestore.doc(`organizations/${orgid}`).get();
 
       return orgDocument.data()
-    } catch (err) {
+    } catch (error) {
       console.error("Error fetching organization", error);
     }
   };
 
+  getMembersOfOrg = async orgid => {
```

Comment:
Can we make `origid` be `organizationID`. In general I think its clearer if we use abbreviations sparingly, and also makes it clear that this `organizationID` corresponds precisely to that in the db.

Reply:
`orgID` should make it clear enough. I'll make the change.

Reply:
Being concise when it's clear. Capitalization should do the trick